pkg/operator/operator_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	fakedynamic "k8s.io/client-go/dynamic/fake"
 	"k8s.io/client-go/informers"
 	fakekube "k8s.io/client-go/kubernetes/fake"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
```

Comment:
nit non-blocking: isn't this normally just imported as scheme? Don't think it clashes so could leave as scheme?